components/angular-tomitribe-tooltip/src/tomitribe-tooltip.ts

```diff
@@ -126,36 +126,43 @@ module tomitribe_fab {
         function calcTooltipPosition() {
             if (!angular.isDefined(tribeTooltip) || !angular.isDefined(tooltip)) return;
 
+            let positions = tribeTooltip.position.split(' ');
+
+            if(positions.length === 1) {
+                positions.push("middle");
+            }
+
             var width = $element.outerWidth(),
                 height = $element.outerHeight(),
                 top = $element.offset().top,
                 left = $element.offset().left;
 
-            if (tribeTooltip.position === 'top')
-            {
-                tooltip.css(
-                    {
-                        left: left - (tooltip.outerWidth() / 2) + (width / 2),
-                        top: top - tooltip.outerHeight()
-                    });
-            }
-            else if (tribeTooltip.position === 'bottom')
+            if (positions[0] === 'top' || positions[0] === 'bottom')
```

Comment:
Please replace `positions[0] === 'top'` with `positions.indexOf('top') > -1` and other compatisons too...

Reply:
We do not expect person to know in which order everything should be, also you can do same thing in strings (array is good too, just use indexOf)